core/templates/core/login.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'core/base.html' %}
+{% block title %}Logowanie{% endblock %}
+
+{% block head %}
+<script type="module">
+    const canvas = document.getElementById('starback-bg');
+        const starback = new Starback(canvas, {
```

Comment:
delete this indent
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/a774e88d-9ff8-4eb8-8f0f-0cea8f40a981)